docs/source/tutorials/evaluate_detections.ipynb

```diff
@@ -25,7 +25,7 @@
     "\n",
     "In this tutorial, we'll use an off-the-shelf [Faster R-CNN detection model](https://pytorch.org/docs/stable/torchvision/models.html#faster-r-cnn) provided by PyTorch.\n",
     "\n",
-    "To use it, you'll need to install `torch` and `torchvision`, if necessary. We'll also need `pycocotools` for evaluation:"
+    "To use it, you'll need to install `torch` and `torchvision`, if necessary."
```

Comment:
So `pycocotools` is not required for evaluation anymore, but please check if it is still required in order to download the COCO dataset from the zoo (per my comment later in the PR)

Reply:
Yeah, this is still required. My bad.